addons/account_edi_ubl_cii/models/account_move.py

```diff
@@ -43,6 +37,12 @@ def _get_ubl_cii_builder_from_xml_tree(self, tree):
                 return self.env['account.edi.xml.ubl_sg']
             if 'urn:cen.eu:en16931:2017' in customization_id.text:
                 return self.env['account.edi.xml.ubl_bis3']
+        ubl_version = tree.find('{*}UBLVersionID')
+        if ubl_version is not None:
+            if ubl_version.text == '2.0':
+                return self.env['account.edi.xml.ubl_20']
+            if ubl_version.text in ('2.1', '2.2', '2.3'):
+                return self.env['account.edi.xml.ubl_21']
```

Comment:
I don't get why you need this diff :thinking:

Reply:
Because I thought the some specific formats among ubl_sg, ubl_de, ubl_bis3, etc... might have an `UBLVersionID`, and would hence be imported as a regular UBL 2.0/UBL 2.1 file. Thus, we would not take the endpoints and EAS into account when decoding the partner.
But in fact, it's never the case: bis3 doesn't contain this tag, so this code is indeed no necessary :sweat_smile: